docs/docs-graphql/schema.graphqls

```diff
@@ -1157,6 +1157,48 @@ The `DirectoryID` scalar type represents an identifier for an object of type Dir
 """
 scalar DirectoryID
 
+"""A definition of a custom enum defined in a Module."""
+type EnumTypeDef {
+  """A doc string for the enum, if any."""
+  description: String!
+
+  """A unique identifier for this EnumTypeDef."""
+  id: EnumTypeDefID!
+
+  """The name of the enum."""
+  name: String!
+
+  """
+  If this EnumTypeDef is associated with a Module, the name of the module. Unset otherwise.
+  """
+  sourceModuleName: String!
+
+  """The values of the enum."""
+  values: [EnumValueTypeDef!]!
+}
+
+"""
+The `EnumTypeDefID` scalar type represents an identifier for an object of type EnumTypeDef.
+"""
+scalar EnumTypeDefID
+
+"""A definition of a value in a custom enum defined in a Module."""
+type EnumValueTypeDef {
+  """A doc string for the enum value, if any."""
+  description: String!
+
+  """A unique identifier for this EnumValueTypeDef."""
+  id: EnumValueTypeDefID!
+
+  """The name of the enum value."""
+  name: String!
+}
+
+"""
+The `EnumValueTypeDefID` scalar type represents an identifier for an object of type EnumValueTypeDef.
+"""
+scalar EnumValueTypeDefID
```

Comment:
Hm something here doesn't look right. `EnumValues` shouldn't have IDs should they?

Should this even be exposed over the API?

Reply:
I don't know, I just followed @helderco API

Reply:
`EnumValueTypeDefID` isn't in @helderco's suggested API (at least the one in the top-level of your PR)